net/unbound/files/unbound.init

```diff
@@ -6,6 +6,9 @@ START=61
 USE_PROCD=1
 
 start_service() {
+	chmod -R u=rwX,g=rX,o= /etc/unbound
+	chown -R unbound:unbound /etc/unbound
+
```

Comment:
Can this part be put into the `postinst` script so that we do not have to do the chmod call on each service start?